### ISSUE TITLE
List the current wallpapers in JSON through a command line option

### ISSUE THREAD

**Issue by nikolaizombie1** [anufrievroman/waypaper#55]
I would like to add a command line option to list the currently set wallpapers and monitors in the form of a JSON printed to standard output.
The usage of this option would look something like this:
```bash
waypaper --list
```
The output of the command would look something like this:
```json
[
    {
        "monitor": "All",
        "wallpaper": "~/Downloads/Wallpaper1.jpeg"
    },
    {
        "monitor": "DP-2",
        "wallpaper": "/home/user/Downloads/Wallpaper2.png"
    },
    {
        "monitor": "HDMI-A-1",
        "wallpaper": "/home/user/Downloads/Wallpaper3.png"
    }
]
``` 
This would address some of the concerns raised in issue #35 in a way that allows the output to be parsed at the command line or by another program and could be a stepping stone towards implementing #35.

**Comment by anufrievroman** [comment]
Sounds good!

**Comment by anufrievroman** [comment]
Okay, so this has been released in version 2.3. Thank again for your contribution!
